src/main/java/com/google/sps/servlets/RoutesStoring.java

```diff
@@ -44,6 +44,22 @@ String getErrorMessage() {
     }
   }
 
+  public enum UserAccessType {
+    OWNER(1),
+    EDITOR(2),
+    VIEWER(3);
+
+    private int numericValue;
+
+    private UserAccessType(int newValue) {
```

Comment:
This constructor should probably not be private if you want to convert from ints to the enum.